test/extensions/common/wasm/test_data/test_cpp/bazel/external/proxy-wasm-cpp-sdk.BUILD

```diff
@@ -0,0 +1,34 @@
+licenses(["notice"])  # Apache 2
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "proxy_wasm_intrinsics",
+    srcs = [
+        "proxy_wasm_intrinsics.cc",
+        "proxy_wasm_intrinsics_lite.pb.cc",
+        "struct_lite.pb.cc",
+    ],
+    hdrs = [
+        "proxy_wasm_api.h",
```

Comment:
Can we get contrib/ files in here?

Reply:
Contrib files should be included as a separate target I would think.

Reply:
That's fine but can you add a BUILD file? It's not visible otherwise.

Reply:
Addressed by using the BUILD file from the repo.  This enables the contrib BUILD file to be accessed directly there.